editstyle.css

```diff
@@ -0,0 +1,111 @@
+*{
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+.container {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+.header {
+    background-color: #333;
+    padding: 15px;
+    text-align: end;
+}
+.header a {
+    color: #fff;
+    text-decoration: none;
+    font-size: 20px;
+}
+.main {
+    display: flex;
+    flex: 1;
+    margin-top: 20px;
+}
+.about{
+    background-color: aquamarine;
+}
+.content {
+    flex: 1;
+    padding: 1rem;
+    margin-right: 20px;
```

Comment:
same as above, However if space is must needed apply padidng instead of margin

Reply:
done